src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -661,8 +661,6 @@ private void recursivelyCopyProjectDependenciesWithScope(
             copiedConf.getAttributes().attribute(Usage.USAGE_ATTRIBUTE, internalUsage);
             // Must set this because we depend on this configuration when resolving unifiedClasspath.
             copiedConf.setCanBeConsumed(true);
-            // But this should never be resolved! (it will most likely fail to given the usage above)
```

Comment:
This is the real change to make it work. You can't alter `resolvable` after configuration has been created. If configuration is resolvable the clone will be and there's no way of changing it.

Reply:
Thanks for the fix! This sounds ok to me (and I guess we have no choice) but GCV are always exciting in unexpected ways. I'm going to release an RC and check this works out fine on a few internal repos.

Reply:
https://github.com/palantir/gradle-consistent-versions/releases/tag/2.14.0-rc1

Reply:
Had to make a branch here so we can make an RC release: https://github.com/palantir/gradle-consistent-versions/pull/1064. Gonna close this PR in favour of that one.